Changelog

```diff
@@ -18,9 +18,10 @@ The full VCS changelog is available here:
 Nibabel releases
 ****************
 
-Most work on NiBabel so far has been by Matthew Brett (MB), Michael Hanke (MH)
-Ben Cipollini (BC), Marc-Alexandre Côté (MC), Chris Markiewicz (CM), Stephan
-Gerhard (SG), Eric Larson (EL), Yaroslav Halchenko (YOH) and Chris Cheng (CC).
+Most work on NiBabel so far has been by Matthew Brett (MB), Chris Markiewicz
+(CM), Michael Hanke (MH), Marc-Alexandre Côté (MC), Ben Cipollini (BC), Paul
+McCarthy (PM), Chris Cheng (CC), Yaroslav Halchenko (YOH), Satra Ghosh (SG),
```

Comment:
That's two SGs now. @satra @unidesigner What are your middle initials?

Reply:
SUG works for me :) Greetings!